### PR TITLE
DELETE Service and test cases for it

### PR DESCRIPTION
**Pull request by gdrdrdr**

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -112,5 +112,35 @@ def test_list_suppliers(self):
...
+        self.assertEqual(len(test_suppliers), 5)
 
+        """Delete a Supplier """
+        resp = self.app.delete('/suppliers/{}'.format(test_suppliers[0].id),
+                               content_type='application/json')
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(len(resp.data), 0)
 
+        """"Check length of db""" 
+        new_count = self.get_supplier_count()
```
[the remark this answers is not in the document]
Comment (gdrdrdr) [reply tests/test_routes.py:128]: function added

[the remark this answers is not in the document]
Comment (snr6450) [reply tests/test_routes.py:128]: I checked just now and delete service is working well. I think we can close this issue.

**Comment by snr6450** [comment]
Delete works as expected!

**Review by snr6450 — COMMENTED** [review]
Looks good

**Pull request merged**